must-gather/Dockerfile.rhel7

```diff
@@ -1,8 +1,8 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
-WORKDIR /go/src/github.com/openshift/ptp-operator
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-7-golang-1.15-openshift-4.6 AS builder
```

Comment:
As above, what is the reason for two different Dockerfiles in this directory?  If this one is meant to be used by CI or ART builds, then the "rhel7" shouldn't be taken literally anymore, and rhel-8-golang-1.15-openshift-4.6 should be used instead.

Reply:
Yes, we shall use rhel-8.